libs/cv_engine.py

```diff
@@ -115,9 +125,21 @@ def __process(self, cv_image):
             self.log_detail["Tracker"].append(tracker_time)
         return cv_image, tmp_objects_list, post_processing_data
 
-    def process_video(self, video_uri):
+    def process_video_file(self, video_uri, video_date: str = None):
         input_cap = cv.VideoCapture(video_uri)
-        fps = max(25, input_cap.get(cv.CAP_PROP_FPS))
+        fps = input_cap.get(cv.CAP_PROP_FPS)
```

Comment:
this is overridden below so you can remove this line

Reply:
Removed